webdriver-spec.html

```diff
@@ -707,9 +707,14 @@
 
 <dl>
  <dt><dfn data-lt="intermediary nodes">Intermediary node</dfn>
- <dd>Intermediary nodes are those that act as proxies,
-  implementing both the client and server sides of the <a href=#protocol>protocol</a>.
-  However they are not expected to implement <a>remote end steps</a> directly.
+ <dd>Intermediary nodes are those that act as proxies, implementing
+  both the <a>local end</a> and <a>remote ends</a> of
+  the <a href=#protocol>protocol</a>.  However they are not expected
+  to implement <a>remote end steps</a> directly. All nodes between a
+  specific <a>intermediary node</a> and a <a>local end</a> are said to
+  be <dfn>downstream</dfn> of that node. Conversely, any nodes between
+  a specific <a>intermediary node</a> and an <a>endpoint node</a> are
+  said to be <dfn>upstream</dfn>.
```

Comment:
Can you add `<dfn data-lt="upstream node">` and `<dfn data-lt="downstream node">` if people want to link to our spec, please?